src/Symfony/Component/HttpClient/Response/CurlResponse.php

```diff
@@ -133,10 +135,6 @@ public function __construct(CurlClientState $multi, $ch, array $options = null,
                     throw $e;
                 }
             }
-
-            curl_setopt($ch, CURLOPT_HEADERFUNCTION, null);
-            curl_setopt($ch, CURLOPT_READFUNCTION, null);
-            curl_setopt($ch, CURLOPT_INFILE, null);
```

Comment:
This is not needed - I've actually seen complains from `curl_multi_exec()` about this state change while processing the responses.